receiver/awsecscontainermetricsreceiver/awsecscontainermetrics/constant.go

```diff
@@ -23,6 +23,51 @@ const (
 	AttributeECSTaskRevesion = "ecs.task-definition-version"
 	AttributeECSServiceName  = "ecs.service"
 
+	CPUsInVCpu               = 1024
+	BytesInMiB               = 1024 * 1024
 	ContainerMetricsLabelLen = 3
 	TaskMetricsLabelLen      = 6
+
+	TaskPrefix                        = "ecs.task."
+	ContainerPrefix                   = "container."
+	ResourceAttributeServiceNameKey   = "service.name"
+	ResourceAttributeServiceNameValue = "awsecscontainermetrics"
```

Comment:
The service name corresponds with an application, not a backend, so for example `AuthService`, `SearchFrontend`, etc. We could fill this in with the ECS service name, or otherwise we shouldn't fill it since this isn't the correct semantics

Reply:
Yea, got it. But, for all of our AWS receivers, we are using the receiver name as `service.name`. Because, this field will be utilized by our CW EMFExporter to generate different rules for different receivers. Especially for Container Insights.

Reply:
What do you mean by AWS receivers? I think the only one we have is xray, which doesn't do this, and definitely shouldn't since we need to make sure the app's service name is used.

I'm not sure what you mean exactly by the rules, but anyways we can't just fill a semantic convention attribute with something that doesn't follow the spec. If anything, the telemetry.sdk matches closer to what this sort of receiver is doing. @bogdandrutu @tigrannajaryan any suggestion on that?

Also @hossain-rayhan it's important to take a step back and remember what this receiver is here for - it's to translate the container metrics data into the OpenTelemetry format / specification. This is because this data seems useful to users regardless of if they use cloudwatch or not. While we may need some, but hopefully not much, consideration for specific vendors like cloudwatch, that's not the intent here. If you haven't yet, you should go through in detail at least the Resource and Metrics semantics conventions of OTel spec before proceeding and make sure you are aligned with it https://github.com/open-telemetry/opentelemetry-specification/tree/master/specification/resource. That doesn't mean we want to block data that's needed, but it's important to follow the spec as much as possible.

Reply:
This receiver generates ECS container Metrics itself but not receiving any metrics from outside of AOC. For the metrics generated inside the receiver, the idea is to put receiver name in ```service.name``` attribute on these metrics. It's similar to the idea Prometheus receiver uses ```job_name``` as ```service.name``` for metrics it scrapes.

Reply:
>I'm not sure what you mean exactly by the rules, but anyways we can't just fill a semantic convention attribute with something that doesn't follow the spec. If anything, the telemetry.sdk matches closer to what this sort of receiver is doing. @bogdandrutu @tigrannajaryan any suggestion on that?

+1. We should not use "service.name" for receiver name. That is not the purpose of "service.name". "service.name" is supposed to describe the source that emits the metrics. Collector is just collecting the metric, it is an intermediary, it is not the source. Nor is "telemetry.sdk" intended for that.

The source that emits the metrics is the container here. If we know the name of the service that runs in the container we should set that. If we don't know we should not record it at all.

I do not know why we want to record the receiver name, perhaps you can clarify the use case. This can then be added as a semantic convention for OpenTelemetry as a whole or just for the Collector and will possibly end up in the "otel" namespace.

Reply:
@hossain-rayhan If you can give some more detail about this usage that would be great. I think filling in wrong information is blocking this PR, so the easiest way to proceed would be to just remove setting the service name for now and we can figure out a way to handle what you need in a separate PR.

Reply:
Thanks @anuraaga . I am removing it for now as this should not block the receiver. This is more related to the exporter logic as it's being utilized for supporting special customer use cases. If needed I can support it in a separate PR after further discussion.